src/react-apps/applications/shared/src/navigation/main-header/appBar.tsx

```diff
@@ -0,0 +1,174 @@
+    flexGrow: 1,
+    zIndex: 1,
+    color: 'black',
+  },
+  appBar: {
+    borderBottom: '1px solid',
+    borderBottomColor: '#C9C9C9',
+  },
+  breadCrumb: {
+    paddingLeft: 30,
+    fontSize: 20,
+  },
+  breadCrumbSubApp: {
+    color: '#0062BA',
+  },
+  button: {
+    border: '2px solid #0062BA',
+    borderRadius: 0,
+    color: '#0062BA',
+    fontSize: 18,
+    padding: '2px 8px 4px',
+    textTransform: 'lowercase',
+  },
+  paper: {
+    textAlign: 'center',
+  },
+  subHeader: {
+    'borderBottom': '2px solid',
+    'paddingLeft': 18,
+    'paddingRight': 18,
+    'paddingBottom': 6,
+    'color': 'black',
+    'fontSize': 20,
+    'borderBottomColor': 'transparent',  // To mitigate the 1 pixel adjustment
+    '&:hover': {
+      borderBottomColor: 'black',
+    },
+  },
+  subHeaderActive: {
+    color: '#0062BA',
+    borderBottomColor: '#0062BA',
+  },
+  topRightService: {
+    paddingRight: 22,
+  },
+});
+
+class AppBarComponent extends React.Component<IAppBarComponentProps, IAppBarComponentState> {
+  public state: IAppBarComponentState = {
+    anchorEl: null,
+    tabletDrawerOpen: false,
+  };
+
+  public handleTabletDrawerMenu = () => {
+    this.setState((state: IAppBarComponentState) => {
+      return {
+        tabletDrawerOpen: !state.tabletDrawerOpen,
+      };
+    });
+  }
+
+  public render() {
+    const { activeLeftMenuSelection, activeSubHeaderSelection, classes, org, service } = this.props;
+    return (
+      <div className={classes.root}>
+        <AppBar
+          position='static'
+          color={this.props.backgroundColor}
+          className={classes.appBar}
+          elevation={0}
+        >
+          <Toolbar>
+            <Grid container={true} direction='row' alignItems='center' justify='space-between'>
+              <Grid xs={true} item={true} container={true}>
+                <Grid item={true}>
+                  <img src='/designer/img/altinn_logo_header.png' />
+                </Grid>
+                <Hidden mdUp>
+                  <Grid item={true} className={classes.breadCrumb}>
+                    / {activeSubHeaderSelection} /
+                      <span className={classes.breadCrumbSubApp}> {activeLeftMenuSelection} </span>
+                  </Grid>
+                </Hidden>
+              </Grid>
+              <Hidden smDown>
+                <Grid xs={true} item={true} className={classes.paper}>
+                  {service != null ? service : 'WARNING: NO SERVICE NAME'}
+                </Grid>
+              </Hidden>
+              <Grid item={true} xs={true} container={true} direction='row' alignItems='center' justify='flex-end'>
+                <Grid item={true}>
+                  <Hidden smDown>
+                    {org != null ? org : 'WARNING: NO ORG'}
+                  </Hidden>
+                  <Hidden mdUp>
+                    <div className={classes.topRightService}>
+                      {service !== null ? service : 'WARNING: NO SERVICE NAME'}
```

Comment:
Should use text from language api

Reply:
Could also be simplified with `!service ? service : [text from api]`

Reply:
Language will be fixed in separate issue.

Replaced code with {service ? service : 'WARNING: NO SERVICE NAME'}